app/views/registers/show.html.haml

```diff
@@ -36,6 +36,9 @@
           %dt Managed by:
           %dd
             = link_to @register.authority, "https://www.gov.uk/government/organisations/#{@register.authority.parameterize}", target: :blank
+          %dt No. of records:
```

Comment:
@bahmady is this abbreviation OK, or would you rather write out the whole word?

Reply:
@danmitchell- you should put a comma in large numbers as per: https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#numbers you can use my code from https://github.com/openregister/registers-frontend/pull/76

Reply:
@gidsg Ideally it should be "Number"

Reply:
Was going off the mockups from @conordelahunty but happy to update 👍

Reply:
GDS guidance on acronyms and abbreviations is a bit unclear, but something like "No." would generally be avoided because it's the sort of thing that's not easy for someone with English as a second language to parse.

Reply:
Updated @bahmady! Thanks for insight 👍